pystac/serialization/identify.py

```diff
@@ -31,6 +31,15 @@ class OldExtensionShortIDs(Enum):
     FILE = "file"
 
 
+class STACType(str, Enum):
```

Comment:
This could be confused with [STACObjectType](https://github.com/stac-utils/pystac/blob/main/pystac/stac_object.py#L14). There should be a docstring to specify why it's different (took me a minute)

Reply:
Yeah, I wasn't really happy about that either! Another option would be to update the values of `STACObjectType` to be correspond to the allowed `"type"` values. We don't use the actual string values anywhere except in `data-files/examples/example-info.csv` and that is easy enough to update.

Implemented this in 6267c779fad119b768b5f7b559d11bd8dc72564e so we can see what it looks like, let me know if you like that approach.

Reply:
I dig it